pkgs/applications/misc/xteddy/default.nix

```diff
@@ -4,12 +4,15 @@ stdenv.mkDerivation rec {
   name = "xteddy-${version}";
   version = "2.2";
   src = fetchzip {
-    url = "http://deb.debian.org/debian/pool/main/x/xteddy/xteddy_${version}.orig.tar.gz";
+    url = "https://deb.debian.org/debian/pool/main/x/xteddy/xteddy_${version}.orig.tar.gz";
     sha256 = "0sap4fqvs0888ymf5ga10p3n7n5kr35j38kfsfd7nj0xm4hmcma3";
   };
   nativeBuildInputs = [ pkg-config makeWrapper ];
   buildInputs = [ imlib2 xorg.libX11 xorg.libXext ];
   makeFlags = [ "LIBS=-lXext" ];
+  preBuild = ''
+    sed -i 's/man 1 xteddy/man 6 xteddy/' xteddy.c
```

Comment:
Shouldn't this be done upstream?

Reply:
Yes, it should! But I'm not familiar with the Debian way of fixing typos.